src/test/java/com/enterprise/eventmanagementsystem/EventmanagementsystemApplicationTests.java

```diff
@@ -1,20 +1,15 @@
 package com.enterprise.eventmanagementsystem;
-import com.enterprise.eventmanagementsystem.dao.EventRepository;
 import com.enterprise.eventmanagementsystem.dto.Event;
 import com.enterprise.eventmanagementsystem.service.IEventService;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.boot.test.mock.mockito.MockBean;
 
 import static org.junit.jupiter.api.Assertions.*;
-import static org.mockito.Mockito.atLeastOnce;
-import static org.mockito.Mockito.verify;
 
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
 import java.util.ArrayList;
-import java.util.LinkedList;
 import java.util.List;
```

Comment:
This page had a lot of different imports that didn't need to be added as of right now.